man/docker.1.md

```diff
@@ -425,6 +425,26 @@ device.
 
 Example use: `docker daemon --storage-opt dm.use_deferred_removal=true`
 
+#### dm.use_deferred_deletion
```

Comment:
@moxiegirl Above documentation looks good. I will modify my patches and also add the same documentation in docs/reference/commandline/daemon.md file. I realize that documentation for
deferred removal is not present in daemon.md file, so I will add that too.

I don't understand the last point and that is recommending to use deferred deletion when mounting the directory of one container into another container. How does one do that? Is it about creating containers which explicitly export volumes and using those volumes in other containers.

I think I want to recommend the deferred deletion in general case. This feature primarily helps with unintentional device leak across mount namespaces. Where docker knows that it is sharing mount points explicitly, we should not allow container removal and block it at higher layers.

This feature is primarily useful when docker thinks that device/mount point is not being used by docker or any of the containers and it is safe to remove container. But there is a chance that device/mount point has leaked into other mount namespaces because somebody did "unshare -m" or because people did special volume mounts (-v /:/host). So this feature is is useful in those scenarios.